imap_processing/decom.py

```diff
@@ -1,5 +1,37 @@
+# Standard
+import logging
+
+from bitstring import ReadError
+
+# Installed
 from space_packet_parser import parser, xtcedef
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
I don't think you need this because you are only logging ERROR below and this will override if anyone else wants something different.